File: 백준/Bronze/2609. 최대공약수와 최소공배수/최대공약수와 최소공배수.py
A, B = map(int, input().split())
mini = min(A, B)
denominator = 0
for i in range(0, mini):
    if A % (mini - i) == 0 and B % (mini - i) == 0:
        denominator = mini - i
        break
multiple = A * (B // denominator)

print(denominator)
print(multiple)

File: 백준/Bronze/2609. 최대공약수와 최소공배수/최대공약수와 최소공배수.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);
        int A = sc.nextInt();
        int B = sc.nextInt();
        sc.close();
        int mini = A < B ? A : B;
        int denominator = 0;
        for(int i = 0; i < mini; i++){
            if(A % (mini - i) == 0 && B % (mini - i) == 0){
                denominator = mini - i;
                break;
            }
        }
        int multiple = A * (B / denominator);
        System.out.println(denominator);
        System.out.println(multiple);
    }
}